### DISCUSSION TITLE
how do i complile imagick with AVIF support?

### DISCUSSION THREAD

**Discussion by edmundspriede** [ImageMagick/ImageMagick#7228] [Development]
### ImageMagick version

7.1.1-30

### Operating system, version and so on

Ubuntu 20

### Description

i can find how to enable AVIF support for compile

**Comment by patterned** [comment]
Apparantly AVIF support in ImageMagick is handled through libheif, which means you should be using --with-heic=yes when you configure..  for example

```
sudo apt install libheif-dev libavif-dev
sudo ./configure --with-heic=yes --with-webp=yes
sudo make -j$(nproc)
sudo make install
sudo ldconfig
```

`grep heic config.log`
to check the configure log and see if the configuration returns
`HEIC              --with-heic=yes			yes`


`magick identify -list format | grep AVIF`
to see if it worked.
